File: Application/DTOs/TaskDto.cs
using Application.Common;

namespace Application.DTOs;

public class TaskDto : BaseDto<int>
{
    public int ProjectId { get; set; }
    public int? AssignedToEmployeeId { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public int StatusId { get; set; }
    public DateTime? DueDate { get; set; }    
}

File: UI/src/app/services/dtos/task.dto.ts
import {EmployeeDto} from "./employee.dto";

export interface TaskDto {
  id: number;
  projectId: number;
  assignedToEmployeeId: number;
  statusId: number;
  title: string;
  description: string;
  dueDate: Date;
  assignedToEmployee: EmployeeDto;
}
